routing/index_graph_starter.cpp

```diff
@@ -62,6 +74,84 @@ m2::PointD const & IndexGraphStarter::GetPoint(RoadPoint const & rp)
   return GetPoint(FindFakeEdge(rp.GetFeatureId()).GetFrom());
 }
 
+void IndexGraphStarter::GetOutgoingEdgesList(TVertexType const & u, vector<TEdgeType> & edges)
+{
+  vector<JointEdge> jes;
```

Comment:
For me it's ok to use short names for locals, especially in small blocks of code.